addons/link_tracker/tests/test_merge_utm_campaign.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tools import mute_logger
```

Comment:
I don't think you use mute_logger.
Your python linter should warn you of such things, make sure it's correctly configured.
(details here: https://github.com/odoo/enterprise/wiki/Welcome)